### PR TITLE
local vs fcu coordinate frame

### PR DESCRIPTION
**Pull request by Stifael**
changed fcu frame to local_frame. assumption made: fcu is body frame and local_frame is fixed frame

### CODE REVIEW

**Comment by LorenzMeier** [comment]
@Stifael Can you comment on the testing status of this?

File: mavros/launch/px4_config.yaml [hunk 1]
```diff
@@ -61,9 +61,9 @@ imu:
 
 # local_position
 local_position:
-  frame_id: "fcu"
+  frame_id: "local_origin"
```
Comment (TSC21) [inline mavros/launch/px4_config.yaml]: `local_origin` or `local_frame`?

File: mavros/launch/px4_config.yaml [hunk 2]
```diff
@@ -61,9 +61,9 @@ imu:
...
   tf:
-    send: false
```
Comment (TSC21) [inline mavros/launch/px4_config.yaml]: Why set it to send the transform by default?

**Comment by TSC21** [comment]
Closing this as stalled.

**Pull request closed without merging**